tests/testthat/test-TealDatasetConnector.R

```diff
@@ -271,7 +271,7 @@ testthat::test_that("csv_dataset_connector attritubes", {
     RACE = c("sth1|sth2", "sth", "sth"),
     stringsAsFactors = FALSE
   )
-  formatters::var_labels(ADSL_ns) <- letters[1:4] # nolint
+  teal.data::formatters_var_labels(ADSL_ns) <- letters[1:4] # nolint
```

Comment:
```suggestion
  teal.data::var_labels(ADSL_ns) <- letters[1:4] # nolint
```

Reply:
Hey @gogonzo GitHub is having issues today as this was update in a commit, that is not visible in PRs summary of edited files lol 1d7d88ebfd4b4334e1c3ed4fdbb0d8a9f600991f - chaning to col_labels anyway

Reply:
7b7ade64166f19ae0aeb670f1b40c2b6d771a11f